home.php

```diff
@@ -0,0 +1,108 @@
+<?php
+  session_start();
+  ini_set("display_error", "1");
+
+ include("session_check.php");
+
+ $obj = new session_check(); 
+ $check_valid_user = $obj->logged_in();
+
+
+  if($check_valid_user)
+  {
```

Comment:
There is no code to execute in if block so its useless.
so why do not it can be written to just handle the else part some how instead of if else block?